src/components/helpform/helpform.jsx

```diff
@@ -0,0 +1,63 @@
+const connect = require('react-redux').connect;
+const PropTypes = require('prop-types');
+const React = require('react');
+const FormattedMessage = require('react-intl').FormattedMessage;
+
+const HelpForm = props => {
+    const prefix = 'https://mitscratch.freshdesk.com/widgets/feedback_widget/new?&widgetType=embedded&widgetView=yes&screenshot=No&searchArea=No&captcha=yes';
+    const title = `formTitle=${props.title}`;
+    const username = `helpdesk_ticket[custom_field][cf_scratch_name_40167]=${props.user.username || ''}`;
+    const agentText = encodeURI(window.navigator.userAgent.replace(';', ' -'));
+    const browser = `helpdesk_ticket[custom_field][cf_browser_40167]=${agentText}`;
+    const formSubject = `helpdesk_ticket[subject]=${props.subject}`;
+    const formDescription = `helpdesk_ticket[description]=${props.body}`;
+    return (
+        <div>
+            <script
+                src="https://s3.amazonaws.com/assets.freshdesk.com/widget/freshwidget.js"
+                type="text/javascript"
```

Comment:
To prevent this script from from holding up rendering in the case that the domain is blocked/throttled, we can add `async`. And add `defer` just in case there are any `onload` handlers in that script (still unclear what this is here for, but there are errors in the console when we remove it).
```suggestion
                src="https://s3.amazonaws.com/assets.freshdesk.com/widget/freshwidget.js"
                type="text/javascript"
                async defer 
```

Reply:
I added them (not the suggestion) because our eslint requires them to come first. ;)